jsonnetlib/k8s.jsonnet

```diff
@@ -304,6 +304,8 @@ cfg {
     envmap={},
     container={},
     volumes=[],
+    volumeMounts=[],
+    securityContext={},
```

Comment:
you can use [`podSpec.securityContext`](https://github.com/theplant/plantbuild/blob/master/example/k8s/tests/test_pod_spec.jsonnet) for this